INFO:Slither:Compilation warnings/errors on /sb/0xa63b528fd782c99b204969c867a3943614344d45.sol:
Warning: Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
   --> /sb/0xa63b528fd782c99b204969c867a3943614344d45.sol:246:5:
    |
246 |     constructor(address payable charityWallet) public {
    |     ^ (Relevant source part starts here and spans across multiple lines).


ERROR:root:Error in /sb/0xa63b528fd782c99b204969c867a3943614344d45.sol
ERROR:root:Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/__main__.py", line 564, in main_impl
    (results, number_contracts) = process(filename, args, detector_classes, printer_classes)
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/__main__.py", line 53, in process
    triage_mode=args.triage_mode)
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/slither.py", line 58, in __init__
    self._analyze_contracts()
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/solc_parsing/slitherSolc.py", line 243, in _analyze_contracts
    self._analyze_first_part(contracts_to_be_analyzed, libraries)
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/solc_parsing/slitherSolc.py", line 290, in _analyze_first_part
    self._parse_struct_var_modifiers_functions(contract)
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/solc_parsing/slitherSolc.py", line 345, in _parse_struct_var_modifiers_functions
    contract.parse_state_variables()
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/solc_parsing/declarations/contract.py", line 227, in parse_state_variables
    var = StateVariableSolc(varNotParsed)
  File "/usr/local/lib/python3.6/dist-packages/slither_analyzer-0.6.1-py3.6.egg/slither/solc_parsing/variables/variable_declaration.py", line 52, in __init__
    self._init_from_declaration(var, var['value'])
KeyError: 'value'

